moveit_core/robot_model/include/moveit/robot_model/planar_joint_model.h

```diff
@@ -84,4 +84,3 @@ class PlanarJointModel : public JointModel
   double angular_distance_weight_;
 };
 }
-}
```

Comment:
I would leave this as it was, to avoid touching things we don't need to touch...